### PR TITLE
WEBUI-599: 10.10 ondemand workflow improvements

### PR DESCRIPTION
**Pull request by richardsd**
This is an alternative version of the 10.10 ondemand workflow https://github.com/nuxeo/nuxeo-web-ui/pull/1356 with multiple jobs for running the pipeline.
Feedback is very welcome 👋 

Screenshot 2021-12-29 at 12 23 24

### CODE REVIEW

**Review by mnixo — DISMISSED** [review]

**Comment by richardsd** [comment]
Screenshot 2021-12-28 at 11 39 16 1

**Review by mnixo — APPROVED** [review]

**Review by semisse — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -84,13 +84,36 @@
 
   <repositories>
     <repository>
-      <id>public</id>
-      <url>http://maven.nuxeo.org/nexus/content/groups/public</url>
+      <id>maven-public</id>
+      <url>https://packages.nuxeo.com/repository/maven-public/</url>
```
Comment (Gabez0r) [inline pom.xml:88]: We just need to double check first that these are reachable by the slaves on legacy CI.

**Comment by richardsd** [comment]
Legacy pipeline https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/1063 ⏳

File: pom.xml (hunk 1 above)
Comment (richardsd) [reply pom.xml:88]: It is working as expected 👌

**Pull request merged**